Rakefile

```diff
@@ -40,3 +40,15 @@ RSpec::Core::RakeTask.new(:spec) do |t|
   t.rspec_opts = '--format documentation'
 end
 
+# rake plugins
+task :plugins do
+  puts 'Registering plugins'
+  # Get name of each plugin from folder name. Require each plugin and register.
+  # At this point, we are assuming the plugin is *not* registered and is
+  # therefore not in the plugins table.
+  Dir['plugins/*'].each do |name|
```

Comment:
Because before I do that I need to capture the name of the plugin that lives at that /*/ level. But I just realized I can split and get the [-2] or [1] position for the name...

will fix.